sticker/src/tensorflow/tensor.rs

```diff
@@ -42,4 +74,34 @@ impl TensorBuilder {
     pub fn inputs(&self) -> &Tensor<f32> {
         &self.inputs
     }
+
+    #[allow(dead_code)]
+    pub fn labels(&self) -> &L {
+        &self.labels
+    }
+}
+
+impl TensorBuilder<LabelTensor> {
+    pub fn add_with_labels(&mut self, input: &[f32], labels: &[i32]) {
+        self.add_input(input);
+
+        let max_seq_len = self.inputs.dims()[1] as usize;
+        let token_embed_size = self.inputs.dims()[2] as usize;
```

Comment:
That's not only the dimensionality of the token embeddings but might also include other embeddings, right?